### DISCUSSION TITLE
Welcome to scikit-sundae Discussions!

### DISCUSSION THREAD

**Discussion by c-randall** [NREL/scikit-sundae#1] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we
build together.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulhprobert** [comment]
Thanks for doing this project.  I had been using a different attempt at interfacing python with Sundials and had no end of trouble.  Your project is well constructed, well documented, and a breath of fresh air!
